For New Contributors: Update Integration Documentation

### Issue you'd like to raise.

Hello LangChain community,

We're always happy to see more folks getting involved in contributing to the LangChain codebase.

This is a good first issue if you want to learn more about how to set up
for development in the LangChain codebase.

## Goal

Your contribution will make it easier for users to use integrations with the newest LangChain syntax

## Context

As you may have noticed, we’ve recently gone to LangChain 0.1. As part of this, we want to update integration pages to be consistent with new methods. These largely include: (a) new methods for invoking integrations and chains (`invoke`, `stream`), (b) new methods for creating chains (LCEL, `create_xyz_..`).

There are a lot of integrations, so we’d love community help! This is a great way to get started contributing to the library as it will make you familiar with best practices and various integrations.

## Set up for development

There are lots of integration notebooks in https://github.com/langchain-ai/langchain/tree/master/docs/docs/integrations. After making changes there, you should run `make format` from the root LangChain directory to run our formatter.

## Shall you accept

Shall you accept this challenge, please claim one (and only one) of the modules from the list
below as one that you will be working on, and respond to this issue.

Once you've made the required code changes, open a PR and link to this issue.

## Acceptance Criteria

- Uses new methods for calling chains (`invoke`, `stream`, etc)
- Uses LCEL where appropriate
- Follows the format outlined below

## Examples

We've gotten started with some examples to show how we imagine these integration pages should look like. The exact format may look different for each type of integration, so make sure to look at the type you are working on:

- LLMs:
    - https://python.langchain.com/docs/integrations/llms/cohere
- Chat Models:
    - https://python.langchain.com/docs/integrations/chat/cohere
- Vectorstores:
    - https://python.langchain.com/docs/integrations/vectorstores/faiss
- Retrievers:
    - https://python.langchain.com/docs/integrations/retrievers/tavily
    - https://python.langchain.com/docs/integrations/retrievers/ragatouille
- Tools:
    - https://python.langchain.com/docs/integrations/tools/tavily_search
- Toolkits:
    - https://python.langchain.com/docs/integrations/toolkits/gmail
- Memory:
    - https://python.langchain.com/docs/integrations/memory/sql_chat_message_history


## Your contribution

Please sign up by responding to this issue and including the name of the module.

### Suggestion:

_No response_

Has anyone had any issues with getting the docs to build? I continuously get this error when running poetry install:
```
The current project could not be installed: No file/folder found for package langchain-monorepo
If you do not want to install the current project use --no-root
```

Additionally, both `make docs_build` & `make api_docs_build` fail. The docs_build fails because of 
```
[ERROR] Error: Invalid sidebar file at "sidebars.js".
These sidebar document ids do not exist:
- langgraph
```
The api_docs_build has quite a few errors. Not sure if I'm missing downloading something critical.

--- a/docs/docs/integrations/tools/pubmed.ipynb
+++ b/docs/docs/integrations/tools/pubmed.ipynb
@@ -14,7 +14,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
+   "id": "3e90e50f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install xmltodict"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
    "id": "c80b9273",
    "metadata": {},
    "outputs": [],
@@ -24,7 +34,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 3,
    "id": "f203c965",
    "metadata": {},
    "outputs": [],
@@ -34,23 +44,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 4,
    "id": "baee7a2a",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "'Published: <Year>2023</Year><Month>May</Month><Day>31</Day>\\nTitle: Dermatology in the wake of an AI revolution: who gets a say?\\nSummary: \\n\\nPublished: <Year>2023</Year><Month>May</Month><Day>30</Day>\\nTitle: What is ChatGPT and what do we do with it? Implications of the age of AI for nursing and midwifery practice and education: An editorial.\\nSummary: \\n\\nPublished: <Year>2023</Year><Month>Jun</Month><Day>02</Day>\\nTitle: The Impact of ChatGPT on the Nursing Profession: Revolutionizing Patient Care and Education.\\nSummary: The nursing field has undergone notable changes over time and is projected to undergo further modifications in the future, owing to the advent of sophisticated technologies and growing healthcare needs. The advent of ChatGPT, an AI-powered language model, is expected to exert a significant influence on the nursing profession, specifically in the domains of patient care and instruction. The present article delves into the ramifications of ChatGPT within the nursing domain and accentuates its capacity and constraints to transform the discipline.'"
+       "'Published: 2024-02-10\\nTitle: circEPB41L2 blocks the progression and metastasis in non-small cell lung cancer by promoting TRIP12-triggered PTBP1 ubiquitylation.\\nCopyright Information: © 2024. The Author(s).\\nSummary::\\nThe metastasis of non-small cell lung cancer (NSCLC) is the leading death cause of NSCLC patients, which requires new biomarkers for precise diagnosis and treatment. Circular RNAs (circRNAs), the novel noncoding RNA, participate in the progression of various cancers as microRNA or protein sponges. We revealed the mechanism by which circEPB41L2 (hsa_circ_0077837) blocks the aerobic glycolysis, progression and metastasis of NSCLC through modulating protein metabolism of PTBP1 by the E3 ubiquitin ligase TRIP12. With ribosomal RNA-depleted RNA seq, 57 upregulated and 327 downregulated circRNAs were identified in LUAD tissues. circEPB41L2 was selected due to its dramatically reduced levels in NSCLC tissues and NSCLC cells. Interestingly, circEPB41L2 blocked glucose uptake, lactate production, NSCLC cell proliferation, migration and invasion in vitro and in vivo. Mechanistically, acting as a scaffold, circEPB41L2 bound to the RRM1 domain of the PTBP1 and the E3 ubiquitin ligase TRIP12 to promote TRIP12-mediated PTBP1 polyubiquitylation and degradation, which could be reversed by the HECT domain mutation of TRIP12 and circEPB41L2 depletion. As a result, circEPB41L2-induced PTBP1 inhibition led to PTBP1-induced PKM2 and Vimentin activation but PKM1 and E-cadherin inactivation. These findings highlight the circEPB41L2-dependent mechanism that modulates the \"Warburg Effect\" and EMT to inhibit NSCLC development and metastasis, offering an inhibitory target for NSCLC treatment.\\n\\nPublished: 2024-01-17\\nTitle: The safety of seasonal influenza vaccination among adults prescribed immune checkpoint inhibitors: A self-controlled case series study using administrative data.\\nCopyright Information: Copyright © 2024 The Author(s). Published by Elsevier Ltd.. All rights reserv'"
       ]
      },
-     "execution_count": 3,
+     "execution_count": 4,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "tool.run(\"chatgpt\")"
+    "tool.invoke(\"What causes lung cancer?\")"
    ]
   },
   {
@@ -78,7 +88,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.11.7"
   }
  },
  "nbformat": 4,

